manage-apps/register.html

```diff
@@ -80,7 +80,7 @@
           <section class="col col-6">
             <label class="label">Website</label>
             <label class="input">
-              <input type="text" name="website" id="website">
+              <input type="text" name="website" id="website" placeholder="https://example.com">
```

Comment:
should we have the example URL be http rather than https? seems like this would be the more common case.